### ISSUE TITLE
Track relevant hyperparameters in wandb

### ISSUE THREAD

**Issue by sfluegel05** [ChEB-AI/python-chebai#14]
# Status Quo
- All hyperparameters are stored in a file which is uploaded to wandb, some are mentioned directly in the run name or the command
- individual hyperparameters appear as columns (e.g. learning rate, `out_dim`, ...), but class pathes, config names or data-related information are missing

# Goal
- Include relevant hyperparameters as distinct parameters for each run (e.g. model class, chebi version, weighting used, loss function)
- Allow to automatically tag runs based on the config used, allow manual tags (e.g. for specific papers)
